src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -25,6 +25,8 @@
 
 import static io.confluent.connect.elasticsearch.DataConverter.BehaviorOnNullValues;
 import static io.confluent.connect.elasticsearch.bulk.BulkProcessor.BehaviorOnMalformedDoc;
+import static org.apache.http.util.TextUtils.isBlank;
```

Comment:
nit: `import org.apache.commons.lang.StringUtils` instead?

Reply:
I don't think we currently pull in apache commons and I didn't think it is worth it to add the dependency just for this method.

Reply:
that's fair, but I'm not sure if this is intended to be a public API though, for the http library. given the complexity of the code, I'd be more inclined to simply make a copy of the function here.

Reply:
I remove the use of this method and replace with simple `isEmpty()` check. The difference is that null and strings containing spaces will not trigger the feature, which I think is fine since our documentation explicitly says empty string.